app/src/main/java/pl/example/android/garbageapp/ui/ActivitySectorGreen.java

```diff
@@ -37,10 +38,13 @@ protected void onCreate(Bundle savedInstanceState) {
         binding.rcSectorTerms.setLayoutManager(verticalLinearLayoutManager);
         binding.rcSectorTerms.setAdapter(sectorTermsAdapter);
 
+        if(checkIfSectorMarkedAsNotification())
+            binding.notificationSwitch.setChecked(true);
+
         //TODO mViewModel observe and bind data to UI
     }
 
-    public void showToast(View view) {
-        Toast.makeText(this, "Przypomnienie ustawione dla STREFY I", Toast.LENGTH_LONG).show();
+    public void setNotification(View view) {
+        super.setNotification(view);
     }
```

Comment:
When you declare setNotification() method as public method in BaseActivitySector, won't it work?

Reply:
Good point. It was my bad day for finding names :)

Reply:
It was worked but it was warning. I found another approach.